Makefile

```diff
@@ -57,13 +57,13 @@ ftest: bin/pytest bin/elastic-ingest
 	connectors/tests/ftest.sh $(NAME) $(PERF8)
 
 run: install
-	bin/elastic-ingest --debug
+	bin/elastic-ingest --log-level=DEBUG
```

Comment:
Both `--log-level=DEBUG` and `--log-level DEBUG` will work, but I think we should follow the same convention in the application to use space instead of `=`

Reply:
Yup agreed, but I also removed --log-level from `run` - it will use the one defined in config file, IMO it's better.

Reply:
We can have `make run-debug` as a shorthand for this command instead, idk if we need tho.